app/index.jsx

```diff
@@ -76,8 +76,6 @@ const requestForget = streamManager.forget.bind(streamManager);
 
 
 class App extends Component {
-    startingLanguage = 'en';
```

Comment:
@shayanbinary pls learn to handle merge conflicts better

Reply:
![screen shot 2018-08-10 at 8 37 23 am](https://user-images.githubusercontent.com/38674882/43938409-a45947fc-9c78-11e8-93ba-114af4b008cf.png)
it's not conflict. this is `eslint` error. also by moving this line down, all works fine for me. if there is anything else, I'll be glad to know, so that it would not happen next time.

Reply:
that's confusing. because we face 2 different rule for one line. in my side its `startingLanguage should be placed after constructor` and in Travis `startingLanguage should be placed before constructor`

Reply:
use `yarn install` and it's fixed.